content/en/docs/setup/independent/install-kubeadm.md

```diff
@@ -150,6 +150,11 @@ kubelet and the control plane is supported, but the kubelet version may never ex
 server version. For example, kubelets running 1.7.0 should be fully compatible with a 1.8.0 API server,
 but not vice versa.
 
+{{< warning >}}
+Kubeadm and Kubernetes components must be [carefully upgraded](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11/),
+and should not be upgraded with other packages. These install instructions exclude all kubernetes packages from system upgrades.
```

Comment:
s/Kubeadm/kubeadm
s/kubernetes/Kubernetes

The warning text is a little confusing. Maybe reverse the order, something like:
"These instructions exclude all Kubernetes packages from any system upgrades. This is because kubeadm and Kubernetes require [special attention to upgrade](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11/)."